### PR TITLE
Medal case whitelist

### PR DESCRIPTION
**Pull request by Keer-Sar**
## About the PR

Makes the captain's medal case only accept medals.
- Adds new tag "Medal" to tags.yml
- Adds tag: Medal to all medal prototypes in medals.yml
- Adds whitelist: tag: Medal to medalcase.yml
## Why / Balance

Stops captains from doing this: 

thisTbhn't

Using a 2x2 item used to hold their entire locker is powergaming. Similar reasons to #27303
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

🆑 
- tweak: Medal Cases now only hold medals

### CODE REVIEW

File: Resources/Prototypes/tags.yml [hunk 1]
```diff
@@ -866,6 +866,9 @@
 - type: Tag
   id: Medkit
 
+- type: Tag
+  id: Medal
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/tags.yml:871]: a before k you nerd

**Comment by IProduceWidgets** [comment]
An alternate solution would be to make the case only accept small items. I don't have a strong opinion on it though.

**Comment by metalgearsloth** [comment]
If it uses itemmapper this is probably better.

**Comment by IProduceWidgets** [comment]
> If it uses itemmapper this is probably better.

It doesn't. But its such a pretty sprite it probably should.

File: Resources/Prototypes/tags.yml (hunk 1 above)
Comment (metalgearsloth) [inline Resources/Prototypes/tags.yml:871]:
```suggestion
- type: Tag
  id: Medal

- type: Tag
  id: Medkit

```

**Pull request merged**